src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -519,121 +516,35 @@ export function fetchConfirmedAppointmentDetails(id, type) {
   };
 }
 
-const UNABLE_TO_KEEP_APPT = '5';
-const VALID_CANCEL_CODES = new Set(['4', '5', '6']);
-
-export function cancelAppointment(appointment) {
+export function startAppointmentCancel(appointment) {
   return {
     type: CANCEL_APPOINTMENT,
     appointment,
   };
 }
 
-const CANCELLED_REQUEST = 'Cancelled';
 export function confirmCancelAppointment() {
   return async (dispatch, getState) => {
     const appointment = getState().appointments.appointmentToCancel;
-    const isConfirmedAppointment =
```

Comment:
Removed all the analytics and error tracking code in here, and left the Redux dispatch logic and service call